tac/project-updates/2025/2025-annual-Cacti.md

```diff
@@ -0,0 +1,90 @@
+The highlight here is that in the Spring of 2024 we've added the wonderful [Rafael Belchior](https://github.com/RafaelAPB) to our ranks and he's been doing an excellent job of advancing our Typescript reference implementation for the IETF's Secure Asset Transfer Protocol (SATP).
+
+[LFX Insights Page for Cacti](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=cacti&routedFrom=Github&bestPractice=false&repository=all&dateFilters=Last%20Year&dateRange=2024-01-01%20to%202024-12-31&compare=PP&granularity=month&hideBots=true)
+
+1. Best Practices Score: The governance repo that we now have received poor scores overall and the average of the two gets shown on the main page. The governance repo itself is a templated repository that doesn't contain production code (or any code for that matter) so ideally those numbers would be excluded from the calculations. Even without the governance repo's scores in the mix, we wouldn't have a perfect score, but the picture would look much more rosy than the initial glance lets the user believe. One more thing that we've struggled with is that when attempting to run a FOSSA scan on the Cacti repository, it crashes with an out-of-memory error on a machine that has 48GB RAM so the size of the mono-repo is making it a bit of a challenge to execute the scans but we are working on solutions for this as well.
+2. Contributor growth has slowed a little bit, partially due to the maintainers having much less time last year to host LFX mentorship projects which is always a steady stream of new contributors for us.
+3. Pull requests are still up compared to the previous year as we are relentlessly pursuing our goals in the codebase.
+
+[OpenSSF Scorecard Page](https://scorecard.dev/viewer/?uri=github.com/hyperledger-cacti/cacti)
+
+- The 6.3 score is less than ideal on scorecard, but we are still in the process of figuring out how to tell scorecard about certain things such as our fuzzer (Nuclei DAST) which the report says we do not have. 
+- Another false-negative in scorecard is the old CII-Best-Practices badge which we do have, but it might've gotten lost in translation at the time when we moved to the new GitHub org from `hyperledger/cacti` to `hyperledger-cacti/cacti`.
+
+## Maintainer Diversity
+
+<mark>_How many maintainers do you have, and which organizations are they from? How has the maintainers and diversity of your maintainers changed in the past year? Has the number of active maintainers increased/decreased? Has the diversity of maintainers increased/decreased? Please include a link to your existing [MAINTAINERS file](../guidelines/MAINTAINERS-guidelines.md) and the MAINTAINERS file from last year (if appropriate). This is a good opportunity to ensure that your MAINTAINERS file is up to date and to retire any maintainers._
+</mark>
+
+- We have **8 maintainers** at present from 4 different organizations: Fujitsu, IBM, Tecnico Lisboa, Accenture.
```

Comment:
Rafael mentioned above seems to be from `Blockdaemon` should this be in the list?

Reply:
@EnriqueL8 My information might be outdated by Rafael told me to put him down as a representative of the university. I think we had that discussion a while back so it's possible he'd prefer Blockdaemon at this point. @RafaelAPB, any preference?

Reply:
Blockdaemon is fine, thanks @petermetz @EnriqueL8